docs/user-documentation/metadata.md

```diff
@@ -86,12 +91,22 @@ Example of the same EDTF dates displayed using little-endian format:
 Example of a valid EDTF value ('1943-05') and an invalid value ('1943 May') with the corresponding error message:
 ![Screenshot of both a valid ("1943-05") and an invalid ("1943 May") EDTF entry. Displays the error message "Could not parse the date 'May 1943' Years must be at least 4 characters long."](../assets/metadata_edtf_invalid.png)
 
-The configuration for the Default EDTF Widget (including options for extra strict date validation, and allowing date intervals and date sets).
+The configuration for the Default EDTF Widget (including options for extra strict date validation, and allowing date intervals and date sets). This configuration can be set per field by clicking the gear icon next to any Default EDTF Widget field at **Administration** >> **Structure** >> **Content types** >> **Repository Item** >> **Manage form display** (admin/structure/types/manage/islandora\_object/form-display)
 ![Screenshot of the configuration screen for the EDTF Widget](../assets/metadata_edtf_widget_settings.png)
 
 Example of how the EDTF formatter settings can change the display of an EDTF value:
 ![Combined screenshots displaying the EDTF default formatter settings, default on top and modified settings below, with an example formatted EDTF value displayed for each.](../assets/metadata_edtf_formatters.png)
 
+The EDTF formatter settings can be changed per field by clicking the gear icon next to any Default EDTF formatter field at **Administration** >> **Structure** >> **Content types** >> **Repository Item** >> **Manage display** (admin/structure/types/manage/islandora\_object/display).
+
+By default, EDTF date values are indexed in Solr as string values. The entered value (not the displayed value) is indexed.
+
+!!! note "Solr note:"
+    The Solr string data type requires the full field value to match the query in order to count as a match. This means that searching for `2014` will not retrieve a record where the recorded date value is `2014-11-02`.
```

Comment:
I feel like this is a tiny bit misleading. It would depend on if you're searching just in the date created field. if you're doing a search in your site, you could have other things indexed such as the rendered entity as full text, which would likely display your date created - and then a search for 2014 would return this item. 
In addition, I'd be curious how you're defining "searching for" - is that directly to solr? because if i go through a search_api_solr view - I don't see a way to even search on a specific field that isn't a fulltext field or isn't just a regular drupal filter. If I go the route of exposing a drupal filter for the date created field - that does do partial matches by default so I am able to return the result with the full date when just searching the year.

Reply:
I get that. In my mind, this section is about specifically the EDTF date fields (which includes searching them directly, sorting on them, etc.) It also hedges with "by default" several times in attempt to describe the basic behavior out of the box (not sure if that includes site search or any other views/etc one might set up.)

I thought in the Open Sandbox meeting earlier this week that I got a verbal confirmation that by default it was set up to search Solr, not Drupal, but I could have gotten that wrong. 

100% open to this being revised for technical accuracy!

Reply:
I agree, but I think what @kspurgin said is really important, at least, when we did any field-based stuff in Solr in 7, it was huge to understanding and being able to use Solr effectively. 

@elizoller uh....  I thought I was just not clever enough to figure out how to make the a fielded solr search in Drupal. Cause... didn't somebody say that they had field based search working?

Reply:
@kspurgin / @rosiel  it is set up to search solr before default not the drupal search but all of the solr things go through search_api_solr config and use views to query and display results to users. so i guess i'm saying i don't understand how to search just in the edtf field. and out of the box - it looks like we ship with a rendered entity field for islandora_object with the search_index display mode - but i don't think thats configured out of the box - where as if you configure that view mode (or whatever view mode you chose to have for the rendered entity indexing) to display the date created field - it'd do a partial match because its a fulltext field.
this is really long winded. what i'm trying to say is - i have no idea how, through the search api solr views, to configure a fielded search on a field that is not a fulltext field. if that is possible - i'd love if someone would chime in. if it is not provided as out of the box behavior - the only way someone could specific search the date created field is if they use solr directly (ie the solr admin panel at :8983/solr... ) or through constructing a query directly to solr via URL (again at wherever you host your solr - not through drupal) or through programmatically constructing a solr query via drupal (this would require custom code, etc).

Reply:
Commenting directly on this thread based on the main thread conversation: the note is technically correct, so to speak, when talking about adding a filter for **that field** _as opposed to_ the existing full-text search which includes the rendered entity (which may or may not use the ISO date formatting to display the field, depending on site configuration). However, I did find that using the "Is between" operator does provide [a work-around for adding a date-specific filter](#issuecomment-748337081) by leveraging the date structure's use of standard string sorting behavior.